esp/src/eclwatch/stub.js

```diff
@@ -143,7 +143,7 @@ define([
 
                 document.title = widget.getTitle ? widget.getTitle() : params.Widget;
 
-                if (sessionIsActive) {
+                if (sessionIsActive > -1) { // -1 denotes infinite timemout
```

Comment:
Is it ok to have // in the stub files (I saw an issue recently where it caused an issue as newlines had been removed - but that may have been stub.htm. I don't know if there is the same concern here?)

Reply:
This was more of a hint to myself stating that if there is a session with greater than -1 don't bother starting the timer because it is an infinite session. Not sure if the linting causes issues here @GordonSmith would it?